src/main/java/com/sinthoras/hydroenergy/HE.java

```diff
@@ -33,23 +33,23 @@
     public static boolean logicalClientLoaded = false;
     public static final String ERROR_serverIdsOutOfBounds = "Server uses invalid waterIds! Server message ignored. " +
             "Please make sure your config \"maxControllers\" is at least as big as the server you are connecting to!";
-    public static final String WARN_clientConfigMissmatchDetected = "HydroEnergy: Configuration missmatch to the server " +
+    public static final String WARN_clientConfigMissmatchDetected = "HydroEnergy: Configuration mismatch to the server " +
             "found! This might crash somewhat randomly. Please talk to your server admin!";
     public static final String blueprintHintTecTech = "To see the structure, use a "+ BLUE + "Tec" + DARK_BLUE + "Tech" + GRAY + " Blueprint on the Controller!";
 
     public static HEPressurizedWater pressurizedWater = new HEPressurizedWater();;
 	public static final HEWaterStill[] waterBlocks = new HEWaterStill[HEConfig.maxDams];
 	public static final int[] waterBlockIds = new int[HEConfig.maxDams];
 	public static ItemStack hydroDamControllerBlock;
-	public static ItemStack hydroPumpBlocks[] = new ItemStack[GT_Values.VN.length];
-    public static ItemStack hydroTurbineBlocks[] = new ItemStack[GT_Values.VN.length];
+	public static ItemStack[] hydroPumpBlocks = new ItemStack[GT_Values.VN.length];
+    public static ItemStack[] hydroTurbineBlocks = new ItemStack[GT_Values.VN.length];
 	
 	public static boolean DEBUGslowFill = false;
 	public static final IGuiHandler guiHandler = new HEGuiHandler();
 
 	// Texture locations
-    public static String damBackgroundLocation = "textures/gui/he_water_config.png";
```

Comment:
Why did the previous texture location result in an issues? I never had any or heard about it. And I don't like to have gui textures in the blocks folder.

Reply:
It's optional but I was getting this:

```
[16:34:56] [Client thread/ERROR]: +=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=
[16:34:56] [Client thread/ERROR]: The following texture errors were found.
[16:34:56] [Client thread/ERROR]: ==================================================
[16:34:56] [Client thread/ERROR]:   DOMAIN hydroenergy
[16:34:56] [Client thread/ERROR]: --------------------------------------------------
[16:34:56] [Client thread/ERROR]:   domain hydroenergy is missing 1 texture
[16:34:56] [Client thread/ERROR]:     domain hydroenergy has 1 location:
[16:34:56] [Client thread/ERROR]:       mod hydroenergy resources at ./../build/libs/hydroenergy-1.7.10-1.0.8.jar
[16:34:56] [Client thread/ERROR]: -------------------------
[16:34:56] [Client thread/ERROR]:     The missing resources for domain hydroenergy are:
[16:34:56] [Client thread/ERROR]:       textures/blocks/textures/gui/he_water_config.png.png
[16:34:56] [Client thread/ERROR]: -------------------------
[16:34:56] [Client thread/ERROR]:     No other errors exist for domain hydroenergy
[16:34:56] [Client thread/ERROR]: ==================================================
[16:34:56] [Client thread/ERROR]: +=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=
```

when doing `./gradlew runClient` - I don't know what or why it's appending .png and adding textures/blocks at the front; perhaps that check is broken?

Reply:
That message is spurious, will revert this and track it down another way